draft-ietf-httpbis-priority.md

```diff
@@ -237,7 +237,7 @@ values if possible. The smaller the value, the higher the precedence.
 The following example shows a request for a CSS file with the urgency set to
 `0`:
 
-~~~ example
+~~~ http-sf-dict
```

Comment:
Lines 240, 279, 497, and 507 are all lists of header fields.  Why is this one a dict and the others lists?

Reply:
oh this is funny, I assumed this was some automagic tool or process that applied the changes but I think it is actually a hangover from an experiment around structured fields. I presume then that Istarted the PR in a dirty repo. I'll just take these out because I think the experiment took us in a different direction.